### ISSUE TITLE
Adding modifier classes to modal

### ISSUE THREAD

**Issue by davleh** [interactive-pioneers/iptools-jquery-modal#19]
We need an option to apply modifier classes on the modal, e.g. modal--without-padding to apply different modal styles.

**Comment by davleh** [comment]
```javascript
it('expected to have modifier classes ' + config.modifierClasses, function() {
  object.attr('href', '#test').trigger('click');
  var modifierClasses = config.modifierClasses.split(' ');
  return expect(object.data(pluginName).getModal().is('.' + modifierClasses.join(', .'))).to.be.ok;
});
```

**Comment by davleh** [comment]
@ain this could be a test against this ability

**Comment by ain** [comment]
Resolved by ab9f2aafab87f56aecaaa60e8b33abe5045c1a09
